config/skills.js

```diff
@@ -654,7 +663,7 @@ module.exports = {
 		},
 		27: { // Savage Strike
 			'*': {
-				noInterrupt: ['27-31'],
+				noInterrupt: [27],
```

Comment:
I'm quite confident that this shouldn't be here since the skill is strictly to cancel itself around the category however not the skill id in particular.